blocks/gmo-program-details/gmo-program-details.js

```diff
@@ -14,12 +14,51 @@ const platformMappings = getMappingArray('platforms');
 export default async function decorate(block) {
     const encodedSemi = encodeURIComponent(';');
     const encodedProgram = encodeURIComponent(programName);
+
+    blockConfig = readBlockConfig(block);
+
     const programQueryString = `getProgramDetails${encodedSemi}programName=${encodedProgram}${encodedSemi}programID=${encodeURIComponent(programID)}`;
-    const programData = await executeQuery(programQueryString);
+    const deliverableQueryString = `getProgramDeliverables${encodedSemi}programName=${encodedProgram}${encodedSemi}programID=${encodeURIComponent(programID)}`;
+
+    // Start fetching both data sets in parallel
+    const programDataPromise = executeQuery(programQueryString);
```

Comment:
@TyroneAEM Can you not add `await` directly here instead of adding in line number #39 and 169?
```
const programDataPromise = await executeQuery(programQueryString);
const deliverablesPromise = await executeQuery(deliverableQueryString);
```

Reply:
Reverted back to const programData = await executeQuery(programQueryString); 
const deliverables = await executeQuery(deliverableQueryString)